guides/concepts/index.md

```diff
@@ -1 +1,100 @@
+That’s analogous to a developer composing a Card, making it available for others to use, and accepting revisions. Someone could build an app _using_ Cardstack, but Cardstack itself is not an app.
+
+### Schema
+
+Schema is a general term to describe the shape that data takes. What attributes come back from an API request? When developing with Cardstack, you may define the schema that influences how data is stored, its relationships, and what it is named when it is used on the front end.
+
+### Indexers, Searchers, and Writers
+
+These packages are responsible for data operations like caching, searching, and modifying.
+They are found within the Cardstack monorepo, and they define the flow of data through a Cardstack environment.
+They are internal to how Cardstack works, but can be helpful to look at when trying to understand how and when data requests are made.
+
+### JSON:API
+
+Cardstack follows [JSON:API](https://jsonapi.org/), which is a popular specification for building APIs in JSON.
+Think of it as "rules to follow" for how data should be serialized.
+For example, if you wrote a data adapter that connected to a generic, non-JSON:API endpoint, your data adapter should rearrange the response contents to follow the JSON:API format.
+Then, the Cardstack environment will be able to understand and use that data automatically.
+
+Here's an example of a JSON:API response to `GET /articles`:
+
+```json
+{
+  "data": [{
+    "type": "articles",
+    "id": "1",
+    "attributes": {
+      "title": "An Introduction to Cardstack"
+    }
+  }, {
+    "type": "articles",
+    "id": "2",
+    "attributes": {
+      "title": "What is Web 3.0?"
+    }
+  }]
+}
+```
+
+## Connecting with JavaScript Concepts
+
+To be able to read the codebase, get a sense of the mental models, and make your own creations using Cardstack, it helps to be familiar with these concepts.
+
+### Classes
+
+You might already be familiar with
+[Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes)
+from your work in JavaScript or other programming languages.
+If you haven't worked much with Classes, take some time to refresh your understanding.
+
+### async/await and Promises
+
+Cardstack makes significant use of JavaScript's [async/await](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function) and [Promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises).
+
+### Data adapters
+
+A lot of Cardstack's data handling concepts are inspired by trends toward flexible APIs, like GraphQL, and architectural decisions of Ember.
+Many developers like GraphQL because you have flexibility in the data layer.
+When it comes to Ember, Cardstack's data adapters and plugins have some mental model similarities to model loading, and `ember-data` adapters and serializers.
+
+### Components
```

Comment:
Should this section be part of the `Card` section above?

Reply:
No, this is under a new heading that connects Cardstack concepts to general web dev concepts.